### PR TITLE
19 2.1

### PR DESCRIPTION
**Pull request by sizzlephizzle**
Think this is the right place.

### CODE REVIEW

**Comment by papampi** [comment]
Nice idea, will make 0miner much lighter
I will remove the other 2 PR

**Comment by LuKePicci** [comment]
We should ask CryptoTrader to give a look at the nicehash dash issue.

**Comment by papampi** [comment]
Need @CryptAtomeTrader44 confirmation for reverting back NICE-* to NICE_*
Added him to collaborators list, waiting for his confirmation

**Comment by cartesien** [comment]
But i've not installed 19.2.1 on my rig yet.

Why sizzlephizzle wants to change this ??? Where is the problem by using  a dash instead of underscore ?

I changed this in WTM_SWITCHER python script file because WTM use a dash.

Did sizzlephizzle changed something about that by substitution in a proposal of code ?

I must make another disk to make this test. We are at the end of week end and i've not enough time to débug an entire new install.

File: 0miner [hunk 1]
```diff
@@ -41,481 +54,345 @@ then
...
-  then
-    HCD='/home/m1/zec/v3_4/miner'
-  fi
-  if [ $EQUIHASH_MINER == "ZM" ]
-  then
-    HCD='/home/m1/zec/zm/latest/zm_miner'
-    screen -dmSL miner $HCD --server $BTG_POOL --user $BTGADDR --pass $MINER_PWD --port $BTG_PORT  --time $ZM_OPTS ;
-  fi
-  if [ $EQUIHASH_MINER == "EWBF" ]
+## IDENTIFY ALGORITHM START
```
Comment (papampi) [inline 0miner:88]: Do we need this algorithm identify?
We already have it in 1bash, just need to modify algo specific OC part

Comment (LuKePicci) [reply 0miner:88]: Got it, I have a nice idea to rework the algo-specific identification in 1bash, moving further towards ultimate OC settings solution. I can change that in my OC policy branch but I need this @sizzlephizzle implementation to be merged.

**Comment by LuKePicci** [comment]
@sizzlephizzle please solve conflicts with master branch, and revert changes in 1bash, or make 2 different PR.

**Comment by LuKePicci** [comment]
The 0miner optimizations signed-off by @sizzlephizzle  has been included in #37.

**Pull request closed without merging**